src/rgw/setup/rgw.py

```diff
@@ -144,15 +144,26 @@ def start(conf: MappedConf, index: str):
 
         Log.info(f'Configure logrotate for {const.COMPONENT_NAME} at path: {const.LOGROTATE_CONF}')
         Rgw._logrotate_generic(conf)
+
         # Before starting service,Verify backend store value=motr in rgw config file.
         Rgw._verify_backend_store_value(conf)
+
+        # Create motr trace & addb stob dirctory.
+        # Collect fid value of motr to create addb value.
+        config_file = Rgw._get_rgw_config_path(conf)
```

Comment:
we can merge line 153 & 154, same way line 156 &157. 
Going forward lets target minimal changes, though still keeping the code readability

Reply:
Ok got it.
156/157 can still be merged